modules/rostests/winetests/xmllite/reader.c

```diff
@@ -537,7 +537,7 @@ static void test_reader_create(void)
     IXmlResolver *resolver;
     IUnknown *input, *unk;
     IXmlReader *reader;
-    DtdProcessing dtd;
+    ULONG_PTR dtd; // enum DtdProcessing
```

Comment:
Aren't enums going to be always `int`? (especially with MSVC)

Reply:
In C they are int, yes.